src/frontend/components/NavBar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+const logo = "/mars-rover-logo3.png";
+import "./NavBar.css";
+
+export default function NavBar() {
+    return (
+        <>
```

Comment:
React tip: Each _component_ (e.g. this function) should return exactly one HTML element. However, there are times when you want to return two HTML elements consecutively (e.g. Home.js). That's when you use the fragment element `<>`.

In this component, you are returning only the `<ul>` element, so you don't need the `<>`!